test/types/records/generic/genericFieldDecl-used.bad

```diff
@@ -1,4 +1,3 @@
-genericFieldDecl-used.chpl:8: In function 'main':
-genericFieldDecl-used.chpl:9: error: default initialization with type 'One()' is not yet supported
+genericFieldDecl-used.chpl:9: error: default initialization with type 'One' is not yet supported
```

Comment:
It looks like this change replaces an error message that was already being generated with a different one.  Is the original error message still generated somewhere, or should it be considered fully replaced by the one you added?

Reply:
There are other .bad files with the old error message:

```
test/classes/ferguson/generic-field/generic-field-record-not-inited3.bad
test/classes/initializers/records/generics/bug-14945.bad
test/classes/initializers/records/generics/reuse-init-instantiation-var.bad
test/classes/initializers/records/generics/reuse-init-new-instantiation-var.bad
test/classes/initializers/records/generics/varArgDefaultValue.bad
```